mayastor/src/bdev/nexus/nexus_label.rs

```diff
@@ -1611,3 +1616,13 @@ impl Aligned for u64 {
         }
     }
 }
+
+fn unique(offsets: &[u64]) -> Option<u64> {
```

Comment:
Could you document this function? It's not clear what the intent is. It's called unique but it returns an optional single integer. I would typically expect a `unique()` function to return a bool or a number of unique valuesin the vector? Or even a collection of only the unique values?

Reply:
Have added a self documenting comment. The function is used to check if all elements of an array (slice) have the same value, and returns "that" value wrapped as an Option - and None is returned if the criteria is not satisfied (ie. empty or not all the same). I'm open to suggestions for a better name.